src/GovUk.Education.ExploreEducationStatistics.Content.Services/Requests/AnalyticsCaptureRequests.cs

```diff
@@ -14,4 +14,6 @@ public record CaptureZipDownloadRequest(
     string ReleaseName,
     string? ReleaseLabel,
     Guid? SubjectId = null,
-    string? DataSetTitle = null) : IAnalyticsCaptureRequestBase;
+    string? DataSetTitle = null,
+    // TODO: FromPage can be made non-optional later
+    string? FromPage = null) : IAnalyticsCaptureRequestBase;
```

Comment:
Would it make sense to keep this as an enum at this point?  I did so on https://github.com/dfe-analytical-services/explore-education-statistics/pull/5883 - you have to just use a `JsonConvert` attribute targeted at `property` level.